src/app/app.component.ts

```diff
@@ -1,13 +1,14 @@
 import { Component, OnInit } from '@angular/core';
-import {ActivatedRoute, ParamMap} from '@angular/router';
+import { ActivatedRoute, ParamMap } from '@angular/router';
 
 @Component({
     selector: 'f8-app',
     templateUrl: './app.component.html'
 })
 
 export class AppComponent implements OnInit {
-    public stackUrl: string = 'https://recommender.api.openshift.io/api/v1/stack-analyses/9f9a8e4c026c4f6590ed42149c97995d';
+    public stackUrl: string =
+    'https://recommender.api.openshift.io/api/v1/stack-analyses/a0cceacbcac54d4b8d6897ce00b726eb';
```

Comment:
Why did you move this to a new line? It is harder to read, at least indent it.

Also, this appears to be 4 space indents. We are using 2 spaces per indent.

Reply:
i moved this to new line to fix the 100 characters per line lint error. Some files have 4 space indent and some 2 space indent. I have kept the same indent as it was for this file.